yarn.lock

```diff
@@ -2506,6 +2506,47 @@
     prop-types "^15.8.1"
     react-is "^18.2.0"
 
+"@mui/x-data-grid-premium@6.0.0-beta.1":
```

Comment:
Shouldn't the `x-data-grid-generator` also be bumped to avoid these changes? 🤔

Reply:
Indeed, done :+1: